workspace-manager-clienttests/src/main/resources/configs/BasicAuthenticated.json

```diff
@@ -11,7 +11,21 @@
       "userJourneyThreadPoolSize": 2,
       "expectedTimeForEach": 5,
       "expectedTimeForEachUnit": "SECONDS"
+    },
```

Comment:
I'd suggest having separate configs for each test and gluing them together in a suite.

Reply:
I'l work with @zloery on organizing them in a later PR.

Reply:
if we want to organize them with one test per config, we should do that in this PR. we can define the suite later, but we might as well break them out now if we're planning to later.